dev/vagrant/conf/post_bootstrap/00-poststart-example.yml

```diff
@@ -0,0 +1,62 @@
+- name: set variables
```

Comment:
Change the name of this file, like we discussed earlier

Reply:
Will do!